Convex_hull_3/include/CGAL/convex_hull_3.h

```diff
@@ -893,8 +936,20 @@ void convex_hull_3(InputIterator first, InputIterator beyond,
   
   clear(polyhedron);
   // result will be a polyhedron
-  internal::Convex_hull_3::ch_quickhull_polyhedron_3(points, point1_it, point2_it, point3_it,
-                                                     polyhedron, traits);
+  P3_iterator minx, maxx, miny, it;
+  minx = maxx = miny = it = points.begin();
+  ++it;
+  
+  
+  //take extreme points to begin with.
+  internal::Convex_hull_::init_iterators(minx, maxx, miny, it, points);
```

Comment:
compilation error in Travis:
https://travis-ci.org/CGAL/cgal/jobs/403155410#L1983

The minimal testing after such a change is running the test suite of Convex_hull_3!